tests/preprocessing/test_xtb_preprocessor.py

```diff
@@ -43,10 +43,7 @@ def test_xtb_smiles_preprocessor(get_2d_smiles_json):
             len(input_["bond"]),
             len(preprocessor.xtb_bond_features),
         )
-        assert input_["mol_xtb"].shape == (
-            1,
-            len(preprocessor.xtb_mol_features),
-        )
+        assert len(input_["mol_xtb"]) == len(preprocessor.xtb_mol_features)
```

Comment:
'mol_xtb' should be a numpy array, right?

Reply:
its a list actually.